WebRoot/js/zimbraMail/mail/view/ZmMailMsgView.js

```diff
@@ -1831,6 +1831,7 @@ ZmMailMsgView.prototype._renderMessageBody1 = function(params, part) {
         // calendar part in ICS format
         else if (ct === ZmMimeTable.TEXT_CAL) {
             content = ZmMailMsg.getTextFromCalendarPart(bp);
+            content = AjxStringUtil.htmlEncode(content);
```

Comment:
If that's only been called once, we can go ahead with this version also considering the point about keeping data pristine from @plobbes .